proto/lndkrpc.proto

```diff
@@ -20,6 +21,7 @@ message PayOfferResponse {
 message GetInvoiceRequest {
     string offer = 1;
     optional uint64 amount = 2;
+    optional string payer_note = 3;
```

Comment:
It's actually been a while since I wrote protos but I'm quite glad they brought back `optional` fields to v3 finally :)